cumulusci.yml

```diff
@@ -2,7 +2,7 @@ project:
     name: OutboundFundsNPSP
     package:
         name: Outbound Funds (npsp)
-        namespace: outfundsnpspext
+        namespace: lh6uuas
```

Comment:
Did you mean to change the naemspace?

Reply:
Ooops!   Great catch.  That was from a test.

Reply:
@seanpat09 Fixed.